Add Alpha Engine (#359)
SteamDatabase/FileDetectionRuleSets: descriptions/Engine.Alpha_Engine.md

> [**DigiPen's**](https://www.digipen.edu.sg/) Alpha Engine is a propiertary in-house engine used by their students.
> Alpha Engine provides functions to create a window, draw graphics, get input, and a few other things.
> By the end of the semester, students must transition away from using the fully-featured Alpha DLL to using a stripped-down one that only provides the window and graphics functions.

Reviewer: ```suggestion
[**DigiPen's**](https://www.digipen.edu.sg/) Alpha Engine is a proprietary in-house engine used by this school's students.
```
I'm not a fan of this one either but at least no need to make it so long
